readme.md

```diff
@@ -127,6 +127,23 @@ Custom promise module to use instead of the native one.
 - Pify has useful options like the ability to handle multiple arguments (`multiArgs`).
 - Pify does not have [magic behavior](https://nodejs.org/api/util.html#util_custom_promisified_functions) for certain Node.js methods and instead focuses on predictability.
 
+#### Why can't a promisified class function access it's own context?
+
+Pify is unable to automatically bind a function to it's original class context. As a workaround, bind the promisified function to the original class instance.
```

Comment:
This makes it seem like a limitation of `pify`, where it's actually a side effect of the way JavaScript works.

It also is not mentioning the alternative (and in my opinion much better) solution of wrapping the instance itself using `pify` and then accessing the methods through the wrapper. This works perfectly without the need to use `bind`, and it's one of the key features of the library which sets it apart from the built in `util.promisify`.

```js
await pify(someClass).someFunction(...);
```

Reply:
Fixed!